android/expoview/src/main/java/host/exp/exponent/kernel/services/sensors/SubscribableSensorKernelService.java

```diff
@@ -12,26 +12,26 @@
 import java.util.Map;
 import java.util.WeakHashMap;
 
-import host.exp.exponent.kernel.ExperienceId;
+import host.exp.exponent.kernel.ExperienceKey;
 
 public abstract class SubscribableSensorKernelService extends BaseSensorKernelService {
   protected static int DEFAULT_UPDATE_INTERVAL = 100;
 
-  private Map<ExperienceId, Integer> mExperienceIdListenersCountMap = new HashMap<>();
+  private Map<ExperienceKey, Integer> mExperienceKeyListenersCountMap = new HashMap<>();
```

Comment:
While this works because the device sensor subscriptions are scoped to the lifetime of the process, scope key is conceptually more appropriate here since a project's subscriptions shouldn't change if the project is renamed or transferred.

(This is a philosophical point right now but if someone were to copy it to a module that retains state across launches, using the ExperienceKey to scope would not be transfer-safe.)